src/server/components/annual-review/controller.ts

```diff
@@ -0,0 +1,145 @@
+import { Request, Response, NextFunction } from "express";
+import { startCase } from "lodash";
+import { Status } from "@prisma/client";
+import { add, isPast } from "date-fns";
+
+import * as Types from "../dashboard/listsItems/types";
+import { findListItemByReference } from "server/models/listItem/listItem";
+import { getDetailsViewModel } from "server/components/dashboard/listsItems/getViewModel";
+import { getCSRFToken } from "server/components/cookies/helpers";
+import { HttpException } from "server/middlewares/error-handlers";
+import { prisma } from "server/models/db/prisma-client";
+
+import type { ListItemGetObject, List } from "server/models/types";
+
+export async function confirmGetController(req: Request, res: Response, next: NextFunction): Promise<void> {
+  try {
+    const { listItemRef } = req.params;
```

Comment:
I don't know if you intended it, but ref was a good shout. It will be very unlikely anyone will be able to guess someone else's ref compared to the `id` (which risks someone changing someone else's application by accident or otherwise..)

Reply:
Yep, it was intentional for security reasons. I was going to generate my own uuid's until I saw ref in the DB and thought to use that instead 😁